CMakeLists.txt

```diff
@@ -22,6 +22,9 @@ cmake_minimum_required(VERSION 3.10.2)
 
 # NONE = this project has no language toolchain requirement.
 project(Vulkan-Headers NONE)
+set(VERSION_STRING "1.1.127")
+
+option(VULKAN_HEADERS_SKIP_INSTALL "Skip install" OFF)
```

Comment:
I think @rbseth did add it because a similar flag was added to SPIRV with the following explanation

> Add new CMake option `SPIRV_HEADERS_SKIP_INSTALL` to avoid installing spirv-headers if it is not required by upper component.

Reply:
Yup, that's the reason. Just following the SPIR-V Headers